docs/api/api.md

```diff
@@ -103,6 +103,7 @@ The [Mbed OS RTOS](rtos.html) capabilities include managing objects such as thre
 <tbody>
 <tr>
 <td><a href="thread.html">Thread</a></td>
+<td><a href="thisthread.html">ThisThread</a></td>
```

Comment:
This goes down a section and pushes the other APIs down. If you'd like, I can make this change in a separate PR.

Reply:
Separate PR is fine with me.